junit-platform-commons/src/main/java/org/junit/platform/commons/support/scanning/package-info.java

```diff
@@ -0,0 +1,5 @@
+/**
+ * Maintained classpath scanning APIs provided by the JUnit Platform.
```

Comment:
I wouldn't make a statement regarding stability because that's covered by the `@API` annotations.

```suggestion
 * Classpath scanning APIs provided by the JUnit Platform.
```

Reply:
Should I remove it from other `package-info.java` files inside `common` as well?

Reply:
I wasn't aware we used this elsewhere. :sweat_smile: But I'd be in favor of removing it everywhere.